CMakeLists.txt

```diff
@@ -147,11 +147,7 @@ set(DCA_LIBS
   cuda_utils
 )
 
-set(SYSTEM_GPU_COUNT 0)
-
 if (DCA_HAVE_CUDA)
-  EXECUTE_PROCESS(COMMAND bash -c "nvidia-smi -L | awk 'BEGIN { num_gpu=0;} /GPU/ { num_gpu++;} END { printf(\"%d\", num_gpu) }'"
```

Comment:
This is quire useful to have to make decisions about which tests to add.

Reply:
no test  depends on  the number of GPUs.